Dockerfile.debian

```diff
@@ -30,7 +30,33 @@ RUN go get -u google.golang.org/grpc
 RUN go get github.com/pkg/errors
 RUN go get github.com/stretchr/testify/mock
 
-COPY . .
+COPY vendor vendor
+COPY .deps .deps
+
+# this will pre-build and cache files in /vendor dir
+RUN go list -v -export $(cat .deps)
+
+COPY ipfs ipfs
+RUN go list -v -export github.com/livepeer/go-livepeer/ipfs
```

Comment:
Just curious why these dirs `ipfs`, `monitor` and `pm` have a `go list` here when the rest aren't. Perhaps add a comment explaining why?

Reply:
added some comments